File: Python/URI/1072.py

caso = int(input())
dentro = 0
fora = 0
num = 0

for i in range(caso):
    num = int(input())

    if (num >= 10 and num <= 20):
        dentro += 1
    else:
        fora += 1

print(str(dentro)+" in")
print(str(fora)+" out")

File: JS/URI/1072.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var caso = parseInt(lines.shift());
var dentro = 0;
var fora = 0;
var num = 0;

for (; caso > 0; caso--) {
    num = parseInt(lines.shift());

    if (num >= 10 && num <= 20)
        dentro += 1;
    else
        fora += 1;
}


console.log(dentro + " in");
console.log(fora + " out");